### ISSUE TITLE
Misaligned toolbar after using tweaks

### ISSUE THREAD

**Issue by Mikau28** [builtbybel/ThisIsWin11#150]
I'm not sure what tool causes this issue to happen. However, I believe it to be any of the tools that change the "Show more options" right click menu to the default windows 10 version.
<img width="200" alt="Screenshot 2021-11-12 083128" src="https://user-images.githubusercontent.com/40307312/163870675-734f1fbf-08a8-4b17-a1de-cd8bfebbfd1e.png">

The reason I beleive changing the context menu causes this is because the guide here causes the same issue:
https://www.pcgamer.com/windows-11-context-menu-fix-right-click/

**Comment by Deninc** [comment]
Yep having the same issues. How can we revert the taskbar fixes?

**Comment by Deninc** [comment]
Actually the fix is here:
https://github.com/builtbybel/ThisIsWin11/issues/94

Just restore the "Small Taskbar and icons" config.
